web/pandas/pdeps/0008-inplace-methods-in-pandas.md

```diff
@@ -348,22 +320,16 @@ DataFrames. Therefore, we decided to keep the `inplace` keyword for this small s
 ### Standardize on the `copy` keyword instead of `inplace`
 
 It may seem more natural to standardize on the `copy` keyword instead of the `inplace` keyword, since the `copy`
-keyword already returns a new object instead of None (enabling method chaining) when it is set to `True`.
+keyword already returns a new object instead of None (enabling method chaining) when it is set to `False`.
 
 However, the `copy` keyword is not supported in any of the values-mutating methods listed in Group 2 above
 unlike `inplace`, so semantics of future inplace mutation of values align better with the current behavior of
 the `inplace` keyword, than with the current behavior of the `copy` keyword.
 
 Furthermore, with the Copy-on-Write proposal, the `copy` keyword also has become superfluous. With Copy-on-Write
 enabled, methods that return a new pandas object will always try to avoid a copy whenever possible, regardless of
-a `copy=False` keyword. Thus, the general proposal is to actually remove the `copy` keyword from the methods where it is
-currently used.
-
-Currently, for methods where it is supported, when the `copy` keyword is `False`, a new pandas object (same
-as `copy=True`) is returned as the result of a method call, with the values backing the object being shared when
-possible. With the proposed inplace behavior, current behavior of `copy=False` would return a new pandas object with
-identical values as the original object(that was modified inplace), which may be confusing for users, and lead to
-ambiguity with Copy on Write rules.
```

Comment:
Ah, OK, rereading it again, I now better understand what we wanted to say ;) Indeed, if we would use copy=False where we now use inplace=True, it would have a different behaviour with CoW compared to the existing cases with a copy=False option. 

Will add it back (and see if I can try to clarify it a bit more).

Reply:
@lithomas1 I added back the paragraph, slightly rewritten in the hope to make it clearer. Can you check that? (also to ensure we have the same understanding what the paragraph is about) See the last commit added in https://github.com/lithomas1/pandas/pull/33